### DISCUSSION TITLE
Taillight Modes (Day/Night Flash vs Solid) & Light Modes (High Visibility vs Trail)

### DISCUSSION THREAD

**Discussion by cmeid** [maca88/SmartBikeLights#39] [General]
I have searched here and on the wider internet, but am struggling to find good information. I basically have two questions I'd like to understand:

1 - Solid vs. Day/Night Flash Taillight Modes

The taillight modes are documented here: https://www8.garmin.com/manuals/webhelp/GUID-C41F445D-457F-447D-88C8-FE286BF157E9/EN-US/GUID-4587F7D3-DEE0-422D-9450-D125E15C4EA0.htm

I am using SmartBikeLights to only activate my Varia light when I brake or when it detects something. Is it better to put it in solid or in day/night flash for it to blink at cars?

(I want it to blink and be as annoying as possible when it detects something. I have a dynamo light that is always on to provide general visibility)

2 - Impact of Light Modes (Trail vs Hi Vis vs Auto)

The light modes are documented here (https://www8.garmin.com/manuals/webhelp/GUID-C41F445D-457F-447D-88C8-FE286BF157E9/EN-US/GUID-73B08487-BA57-4EF0-A253-D226E229BC68.html). 

Does anyone know if those further modify the taillight modes that are set by SmartBikeLights. @maca88 are you able to set any of those modes as well?

**Comment by maca88** [comment]
1. I think you can always use "Solid" mode. With RTL515, even in "Solid" mode, the light will start flashing when a vehicle approaches you. After the vehicle passes you the light will go back to "Solid". This is something that cannot be overridden by a CIQ application.

2. Smart Bike Lights has three control modes. If you want to use the Garmin built-in modes (Trail, Hi Vis, Auto), you must switch to "Network" control mode. The problem with the built-in modes is that they cannot be modified. For example, the "Auto" mode will use High mode for the headlight in the dark, and there is no option to set it to Medium or Low. When using Bontrager RT Pro as a headlight, the High mode (1300LM) is too bright for me. That was the main reason why I started this project. I wanted to fine-tune the configuration to my liking.
